File: Oxiyopal.Models/ViewModels/HistorialCilindro.cs
using Oxiyopal.Models.Base;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Oxiyopal.Models.ViewModels
{
    public class HistorialCilindro : EntityBase<int>
    {
        public int CilindroId { get; set; }
        public Cilindro Cilindro { get; set; }

        public int UbicacionId { get; set; }
        public Ubicacion Ubicacion { get; set; }

        public bool EsUbucacionActual { get; set; }
    }
}

File: Oxiyopal.WebUI/ClientApp/src/app/models/ViewModels/HistorialCilindro.ts
import { EntityBase } from '../Base/EntityBase';
import { Cilindro } from '../Cilindro';
import { Ubicacion } from '../Ubicacion';

export class HistorialCilindro extends EntityBase<number> {
    cilindroId: number;
    cilindro: Cilindro;

    ubicacionId: number;
    ubicacion: Ubicacion;

    esUbucacionActual: boolean;
}

